src/cmd/vm.rs

```diff
@@ -239,7 +242,13 @@ fn run_start(args: &ArgsStart) -> Result<()> {
     } else {
         run_betty_start(args)?;
 
-        println!("To connect the betty instance, run `lium dut shell --dut localhost:9222`.");
+        info!("Adding the VM instance to DUT list...");
```

Comment:
I think it's a good idea to refactor out the `--add` logic in `lium dut list` (maybe into src/dut.rs or somewhere) and call the same function to do this, to avoid duplicated logic.

Reply:
Got it! I found there was an unused function `register_dut()` in src/dut.rs so I used that instead.